### PR TITLE
[FSTORE-888] Upgrade to flink 1.17

### PR DESCRIPTION
**Pull request by davitbzh**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -5,15 +5,14 @@
 default['flink']['user_id']                          = '1510'
 default['flink']['user-home']                        = "/home/#{node['flink']['user']}"
 
-default['flink']['version_base']                     = "1.14"
-default['flink']['version_bugfix']                   = "3.1"
+default['flink']['version_base']                     = "1.17"
+default['flink']['version_bugfix']                   = "1.0"
```
Comment (SirOibaf) [inline attributes/default.rb:9]: I can't find the branch for this build in the `logicalclocks/flink` repository.

File: attributes/default.rb [hunk 2]
```diff
@@ -44,3 +43,9 @@
 default['flink']['parallelization']['degree']        = node['cpu']['total']
 default['flink']['webclient_port']                   = 8888
 default['flink']['taskmanager']['network_num_buffers']  = 2048
+
+#
+# Featurestore dependencies
+#
+default['flink']['hsfs']['version']                  = node['install']['version']
+default['flink']['hsfs']['url']                      = "#{node['download_url']}/hsfs/#{node['flink']['hsfs']['version']}/hsfs-flink-#{node['flink']['hsfs']['version']}.jar"
```
Comment (SirOibaf) [inline attributes/default.rb:51]: I have a deja-vu here. I thought we discussed this when we merged hsfs-flink to not include it in the distribution as in any case users need to have the dependency on their maven build.

Also I don't see this attribute being used anywhere.

**Review by SirOibaf — CHANGES_REQUESTED** [review]

File: attributes/default.rb (hunk 2 above)
Comment (davitbzh) [reply attributes/default.rb:51]: ahh sorry, leftovers

File: attributes/default.rb (hunk 1 above)
Comment (davitbzh) [reply attributes/default.rb:9]: done https://github.com/logicalclocks/flink/pull/7

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**